luda-editor/client/src/pages/sequence_edit_page/loaded/components/character_editor/render/part_picker.rs

```diff
@@ -24,7 +27,7 @@ impl CharacterEditor {
         cg_file: &CgFile,
         project_id: Uuid,
         cut_id: Uuid,
-        graphic_index: usize,
+        graphic_index: Uuid,
```

Comment:
It's important to keep index in Uuid because index would be different between before and after run action. I'm not that much sure about it but I think Unique index will solve a lot of problem. 
Cons is that it would be tiresome to get item using Uuid, like Random Access will be imposibble unlike Vector's usize random access.

Reply:
Random access is not that important at least in editor.